scylla-cdc/src/e2e_tests.rs

```diff
@@ -1,22 +1,25 @@
 #[cfg(test)]
 mod tests {
-    use crate::cdc_types::ToTimestamp;
-    use crate::consumer::*;
-    use crate::log_reader::CDCLogReader;
-    use anyhow::{anyhow, Result};
-    use async_trait::async_trait;
-    use itertools::Itertools;
-    use scylla::frame::response::result::CqlValue;
-    use scylla::frame::value::{Value, ValueTooBig};
-    use scylla::prepared_statement::PreparedStatement;
-    use scylla::{Session, SessionBuilder};
     use std::collections::{HashMap, VecDeque};
     use std::convert::identity;
     use std::hash::Hash;
     use std::sync::Arc;
     use std::time;
+
```

Comment:
The second commit should be sent in a separate PR. It's a refactor unrelated to the original problem this PR is supposed to solve.